src/types/UserType.ts

```diff
@@ -0,0 +1,29 @@
+export interface UserType {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: Address;
+  phone: string;
+  website: string;
+  company: Company;
+}
+
+interface Address {
+  street: string;
+  suite: string;
+  city: string;
+  zipcode: string;
+  geo: Geo;
+}
+
+interface Geo {
+  lat: string;
+  lng: string;
+}
+
+interface Company {
+  name: string;
+  catchPhrase: string;
+  bs: string;
+}
```

Comment:
Hmm, I'm the only one who didn't describe these interfaces that we don't use)) Great job anyway. Well done